### PR TITLE
add bindMessage prop to goji Webview component

### PR DESCRIPTION
**Pull request by TakeharuKa**
The pr is to add the necessary prop

### CODE REVIEW

File: packages/core/src/components/componentsTypes.ts [hunk 1]
```diff
@@ -811,6 +811,7 @@ export interface OfficialAccountProps extends BaseProps {
 
 export interface WebViewProps extends BaseProps {
   src?: string;
+  bindMessage?: (e: any) => void;
```
Comment (TakeharuKa) [inline packages/core/src/components/componentsTypes.ts:814]: Uses camelCase

Comment (malash) [reply packages/core/src/components/componentsTypes.ts:814]: You should use `onMessage`, and could you please also add `onLoad` and `onError` ?

Ref: https://developers.weixin.qq.com/miniprogram/dev/component/web-view.html

Comment (TakeharuKa) [reply packages/core/src/components/componentsTypes.ts]: @malash Updated.

**Review by malash — APPROVED** [review]
LGTM

**Pull request merged**